kmehr/src/main/kotlin/org/taktik/icure/services/external/rest/v1/controllers/be/KmehrController.kt

```diff
@@ -435,7 +435,7 @@ class KmehrController(
         @PathVariable patientId: String,
         @RequestParam language: String,
         @RequestParam recipientSafe: String,
-        @RequestParam(defaultValue = "0") version: Int,
+        @RequestParam(required = false, defaultValue = "0") version: Int,
```

Comment:
I read mattermost, if defaultValue is not recognized put it required false and not nullable may not work either.

```suggestion
        @RequestParam(required = false, defaultValue = "0") version: Int = 0,
```

Reply:
From what I remember  "= 0" does not work. In every other instance we use @RequestParam(required = false, defaultValue = "0"). But you are right... Tests will be needed.